### PR TITLE
#162725216: Integrate sass and css into the application

### PR DESCRIPTION
**Pull request by mariamiah**
#### What does this PR do?
* Sets up SASS and CSS processing on the client
#### Description of Task to be completed?
* Install and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client
#### How should this be manually tested?
* fetch this branch by running `git fetch origin ch-sass-css-integration-162725216` in your terminal
* check out into the branch using `git checkout ch-sass-css-integration-162725216`
* run `yarn install` to install the dependencies
* run `yarn start` to start the development server
* navigate to localhost:8000
* view the sass effect on the different page elements
#### What are the relevant pivotal tracker stories?
[ #162725216](https://www.pivotaltracker.com/n/projects/2232225/stories/162725216https://www.pivotaltracker.com/n/projects/2232225/stories/162725216https://www.pivotaltracker.com/n/projects/2232225/stories/162725216)

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+src/assets/
```
Comment (MuhweziDeo) [inline .eslintignore]: add a white space

Comment (mariamiah) [reply .eslintignore]: done

**Review by MuhweziDeo — DISMISSED** [review]
Looks good to me

**Review by calvinpete — DISMISSED** [review]
LGTM

**Review by d-rita — DISMISSED** [review]
LGTM

**Review by MuhweziDeo — APPROVED** [review]
looks good

**Review by kdaglas — APPROVED** [review]
lgtm

**Review by malfahad — APPROVED** [review]

**Pull request merged**